packages/core/src/routes-me/user.ts

```diff
@@ -71,6 +70,7 @@ export default function userRoutes<T extends AuthedMeRouter>(
   router.get('/custom-data', async (ctx, next) => {
     const { id: userId } = ctx.auth;
     const user = await findUserById(userId);
+    assertThat(!user.isSuspended, new RequestError({ code: 'user.suspended', status: 401 }));
```

Comment:
i think the `isSuspended` logic can be put into `findUserById()` (more generally, in user queries factory) for cross-api alignment, e.g. define a new `findActiveUserById()`